randovania/layout/base/standard_pickup_state.py

```diff
@@ -67,10 +74,14 @@ def check_consistency(self, pickup: StandardPickupDefinition):
                 f" got {self.num_shuffled_pickups}. ({pickup.name})"
             )
 
-        if pickup.must_be_starting:
+        if pickup.starting_condition == StartingItemBehavior.MUST_BE_STARTING:
             if not self.num_included_in_starting_pickups:
                 raise ValueError(f"Required items must be included in starting items. ({pickup.name})")
 
+        if pickup.starting_condition == StartingItemBehavior.CAN_NEVER_BE_STARTING:
+            if self.num_included_in_starting_pickups:
+                raise ValueError(f"{pickup.name} cannot be a starting item.")
```

Comment:
could you also add a test for `check_consistency()` that uses `with pytest.raises(ValueError, pattern):` to test these failure cases?